aws/resource_aws_ec2_client_vpn_network_association.go

```diff
@@ -0,0 +1,136 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceAwsEc2ClientVpnNetworkAssociation() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsEc2ClientVpnNetworkAssociationCreate,
+		Read:   resourceAwsEc2ClientVpnNetworkAssociationRead,
+		Delete: resourceAwsEc2ClientVpnNetworkAssociationDelete,
+		Importer: &schema.ResourceImporter{
```

Comment:
This import support is missing:
* Acceptance testing in the form of a `TestStep` that includes `ImportState: true` and `ImportStateVerify: true`
* Documentation in the form of a `## Import` section in the resource documentation.

The support should be fully implemented or removed from the pull request. 👍 

If you are opting to keep the support, you will notice that you will need to either implement a custom import function to that calls `d.Set("client_vpn_endpoint_id", ...)` and trims that part of the ID for `d.SetId()` (then use `ImportStateIdFunc` in acceptance testing) or switch the resource to implementing a "complex" ID that includes both IDs in its string, e.g. `ENDPOINT/ASSOCIATION`. For new implementations, we have generally preferred the latter form where we parse the complex ID and continue to use `ImportStatePassthrough`, but either form is acceptable.

Reply:
Oh, this is cruft from my copy/paste when I started working on this.  I'm going to opt to *not* include import functionality here since I wasn't planning on adding it initially.  I can be swayed to add it however because I can see the need for it.